packs/feat-effects/effect-familiar-of-restored-spirit.json

```diff
@@ -0,0 +1,41 @@
+{
+    "_id": "twvhTLrLEfr7dz1m",
+    "img": "systems/pf2e/icons/actions/Passive.webp",
+    "name": "Effect: Familiar of Restored Spirit",
+    "system": {
+        "description": {
+            "value": "<p>When you Cast or Sustain a hex, one willing creature within 15 feet of your familiar gains temporary Hit Points equal to 2 + half your level, which last until the start of your next turn.</p>"
```

Comment:
Consider adding a Granted by.

Reply:
I can do that. I didn't for the last one I did, so I figured it wasn't necessary, but I'll just go back and add it.